File: Python Excel Examples/RangesApiDemo/updateRangeStyle.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.ranges_api.RangesApi(configuration)

name = "UpdateRangeStyle.xlsx"
folder = "Ranges"
sheetName = "Sheet1"
range = "C2:D6"
storage = ""
style = spirecloudexcel.models.Style()
font = spirecloudexcel.models.Font()
font.color = spirecloudexcel.models.Color(0, 160, 32, 240)
font.underline = "Single"
font.size = 13
font.is_italic = True
font.is_bold = True
font.name = "楷体"
style.font = font
borders = []
topBorder = spirecloudexcel.models.Border("Medium", spirecloudexcel.models.Color(255, 255, 0, 0), "EdgeTop")
rightBorder = spirecloudexcel.models.Border("DashDot", spirecloudexcel.models.Color(255, 0, 255, 0), "EdgeRight")
borders.append(topBorder)
borders.append(rightBorder)
style.border_collection = borders
style.indent_level = 2
style.horizontal_alignment = "Left"
api.update_range_style(name, sheet_name=sheetName, range=range, style=style, folder=folder,storage=storage)

File: JavaScript Excel Examples/RangesApiDemo/updateRangeStyle.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudExcel = require('../../src/index');
    var apiClient = new SpirecloudExcel.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudExcel.RangesApi(apiClient);
    
    var name = "updateRangeStyle.xlsx";
    var opts = { folder: "input" };
    var sheetName = "Sheet1";
    var range = "C2:D6";
    var style = new SpirecloudExcel.Style();
    style.Font = new SpirecloudExcel.Font();
    style.Font.Color = new SpirecloudExcel.Color();
    style.Font.Color.A = 0;
    style.Font.Color.R = 160;
    style.Font.Color.G = 32;
    style.Font.Color.B = 240;
    style.Font.Underline = "single";
    style.Font.Size = 13;
    style.Font.Name = "Arial";
    style.Font.DoubleSize = 2;
    style.Font.IsItalic = true;
    style.Font.IsStrikeout = true;
    style.Font.IsBold = true;

    style.BorderCollection = [new SpirecloudExcel.Border(), new SpirecloudExcel.Border()];
    style.BorderCollection[0].LineStyle = "Medium";
    style.BorderCollection[0].Color = new SpirecloudExcel.Color();
    style.BorderCollection[0].Color.A = 255;
    style.BorderCollection[0].Color.R = 255;
    style.BorderCollection[0].Color.G = 0;
    style.BorderCollection[0].Color.B = 0;
    style.BorderCollection[0].BorderType = "EdgeTop";
    
    style.BorderCollection[1].LineStyle = "DashDot";
    style.BorderCollection[1].Color = new SpirecloudExcel.Color();
    style.BorderCollection[1].Color.A = 255;
    style.BorderCollection[1].Color.R = 0;
    style.BorderCollection[1].Color.G = 255;
    style.BorderCollection[1].Color.B = 0;
    style.BorderCollection[1].BorderType = "EdgeRight";
    style.IndentLevel = 2;
    style.HorizontalAlignment = "left";
    
    instance.updateRangeStyle(name, sheetName, range, style, opts, 
      function (error, data, response) {
        if (error) throw error;
    });
})();